spec/aerospike/query_spec.rb

```diff
@@ -512,6 +512,20 @@
 
     end # context
 
+    context "Run queries in the background" do
+      it "executes a put operation asynchronously" do
+        stmt = Aerospike::Statement.new(@namespace, @set)
+        bin5 = Aerospike::Bin.new("bin5", "string_value")
+        ops = [
+          Aerospike::Operation.put(bin5)
+        ]
+        async_task = client.query_execute(stmt, ops)
```

Comment:
I'd rather this be called execute_task. No reason to convey the client has an async capability that it doesn't.

Reply:
Have changed this.